package.json

```diff
@@ -11,6 +11,7 @@
 		"pretest": "npm run build",
 		"test": "mocha && npm run lint",
 		"lint": "eslint --ext .js,.ts .",
+		"format": "prettier --write .",
```

Comment:
similarly, this should be `eslint --fix .`, rather than using prettier to do it

Reply:
That exits with an error when there are non-formatting-related linting errors, and so is unsuitable for a command named `format`.